pyproject.toml

```diff
@@ -11,10 +11,12 @@ shapely = "*"
 numpy = "*"
 scikit-image = "*"
 scipy = "*"
+pydicom = "*"
```

Comment:
pydicom has breaking interface from 1 to 2 that could be problematic, and I imagine same with 3 when it comes.
I have no complaints about them breaking interface because they follow semver, but I imagine things might get complicated.

Reply:
Yup, that's what the lock file is for.

Reply:
I already had to drop the review approval due to a merge conflict (sorry about that).

As such, I took the opportunity to add the following comment to address this concern:

https://github.com/RadiotherapyAI/RadiotherapyAI/blob/85229d5489a5a670abf5e8f95ce50fec62274fdd/pyproject.toml#L8-L25

Let me know if that helps. If there are known breaking versions, that are known to be incompatible, then yes... they're worth excluding. But I don't think there is anything I have used here that is strictly incompatible with older versions of pydicom (that I know of).